tools/lib/html_branches.py

```diff
@@ -119,7 +120,6 @@ def html_branches(text, fn=None):
 
     tree = html_tag_tree(text)
     branches = []  # type: List[HtmlTreeBranch]
-
```

Comment:
actually that must have changed when i restored the walk function to original by copying.
i will change it to back to how it was